radio/src/targets/horus/backlight_driver.cpp

```diff
@@ -67,6 +67,13 @@ void backlightInit()
 
 void backlightEnable(uint8_t dutyCycle)
 {
+#if (0 >= BACKLIGHT_LEVEL_MIN)
+#error BACKLIGHT_LEVEL_MIN MUST >= 0
+#endif
+  if (0 == dutyCycle)
+    BL_TIMER->BDTR &= ~TIM_BDTR_MOE;
+  else if (0 == (BL_TIMER->BDTR & TIM_BDTR_MOE))
```

Comment:
This would never be called since you defined BACKLIGHT_LEVEL_MIN  as 5

Reply:
In 480x272/radio_setup.cpp, 384
-        g_eeGeneral.blOffBright = editSlider(RADIO_SETUP_2ND_COLUMN, y, event, g_eeGeneral.blOffBright, BACKLIGHT_LEVEL_MIN, BACKLIGHT_LEVEL_MAX, attr);
+        g_eeGeneral.blOffBright = editSlider(RADIO_SETUP_2ND_COLUMN, y, event, g_eeGeneral.blOffBright, BACKLIGHT_LEVEL_MIN - BACKLIGHT_LEVEL_MIN, BACKLIGHT_LEVEL_MAX, attr);

Reply:
That's dirty.